bsyncviewer/models/bedes_models.py

```diff
@@ -4,10 +4,10 @@
 
 
 class BedesTerm(models.Model):
-    content_uuid = models.CharField(max_length=100, unique=True)
-    term = models.CharField(max_length=100)
-    category = models.CharField(max_length=100)
-    url = models.CharField(max_length=300)
+    content_uuid = models.CharField(max_length=200, unique=True)
+    term = models.CharField(max_length=200)
+    category = models.CharField(max_length=200)
+    url = models.CharField(max_length=400)
```

Comment:
nice! We should really keep these multiples of base 2, just because, haha. 256, 512, etc. next time, 🤓